compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/PlatformContext.skiko.kt

```diff
@@ -110,6 +115,10 @@ interface PlatformContext {
     val parentFocusManager: FocusManager get() = EmptyFocusManager
     fun requestFocus(): Boolean = true
 
+    fun createPlatformDragAndDropManager(): PlatformDragAndDropManager {
```

Comment:
```suggestion
    val dragAndDropManager: PlatformDragAndDropManager
```
- to be consistent it should be a property, and creation/returning existing are implementation details. Unless it is stateful, and we want a new state every time.
- the name shouldn't contain `platform`, as the owner class already contains it.

Reply:
> to be consistent it should be a property, and creation/returning existing are implementation details. Unless it is stateful, and we want a new state every time.

It is stateful and a new instance is needed every time. Several `RootNodeOwner`s can share the same `PlatformContext`, but each `RootNodeOwner` needs its own `DragAndDropManager`.

Reply:
> the name shouldn't contain platform, as the owner class already contains it.

I think that's confusing even more, but ok.